libs/blocks/faas-config/faas-config.js

```diff
@@ -31,14 +31,14 @@ const getHashConfig = () => {
 
   const encodedConfig = hash.startsWith('#') ? hash.substring(1) : hash;
   let hashConfig = parseEncodedConfig(encodedConfig);
-  hashConfig.pjs36 = hashConfig.p.js[36] || '';
-  hashConfig.pjs39 = hashConfig.p.js[39] || '';
-  hashConfig.pjs92 = hashConfig.p.js[92] || '';
-  hashConfig.pjs93 = hashConfig.p.js[93] || '';
-  hashConfig.pjs94 = hashConfig.p.js[94] || '';
-  hashConfig[149] = hashConfig.p.js[149] || '';
-  hashConfig[172] = hashConfig.p.js[172] || '';
-  hashConfig.q103 = hashConfig.q[103]?.c || {};
+  hashConfig.pjs36 = hashConfig.pjs36 || hashConfig.p.js[36] || '';
```

Comment:
Nit: You could use use OR assignment
```suggestion
  hashConfig.pjs36 ||= hashConfig.p.js[36] || '';
```